workspaces/ballerina/inline-data-mapper/src/components/Diagram/utils/constants.ts

```diff
@@ -34,6 +34,9 @@ export const GAP_BETWEEN_FIELDS = 1;
 
 export const ISSUES_URL = "https://github.com/wso2/product-ballerina-integrator/issues";
 
+export const AUTO_MAP_TIMEOUT_MS = 40000;
+export const AUTO_MAP_IN_PROGRESS_MSG = "Generating inline mappings for your transformation";
```

Comment:
```suggestion
export const AUTO_MAP_IN_PROGRESS_MSG = "Generating mappings for your transformation";
```
Let's avoid mentioning "inline" here, as it's an internal implementation detail that users don't need to be aware of.